topostats/validation.py

```diff
@@ -229,7 +230,17 @@ def validate_config(config: dict, schema: Schema, config_type: str) -> None:
                     error="Invalid value in config for 'plotting.style', valid values are 'topostats.mplstyle' or None",
                 ),
             ),
-            "save_format": str,
+            "savefig_format": Or(
+                None,
+                str,
+                error="Invalid value in config for plotting.savefig_format" "must be a value supported by Matplotlib.",
+            ),
+            "savefig_dpi": Or(
+                None,
+                "figure",
+                int,
```

Comment:
To follow as the same as below (and is a tad more specific)
```suggestion
                lambda n: n > 0,
```